frontend/src/ui/Blocks.tsx

```diff
@@ -57,12 +57,77 @@ const TextBlock: React.FC<TextProps> = ({ title, content }) => (
     </Block>
 );
 
-const VideoListBlock: React.FC<{ title: string | null }> = ({ title }) => (
-    <Block>
-        <Title title={title} />
-        <i>not yet implemented ☹️</i>
-    </Block>
-);
+const VideoListBlock: React.FC<{ title: string | null }> = ({ title }) => {
+    const [THUMB_WIDTH, THUMB_HEIGHT] = [16, 9].map(x => x * 13);
+
+    return (
+        <Block>
+            <Title title={title} />
+            <div css={{
+                display: "flex",
+                flexWrap: "wrap",
+            }}>
+                {DUMMY_VIDEOS.map(v => {
+                    const url = `/v/${v.id}`;
+
+                    return (
+                        <div
+                            key={v.id}
+                            css={{
+                                margin: "12px 8px",
+                                width: THUMB_WIDTH,
+                                "& a": { color: "black", textDecoration: "none" },
+                            }}
+                        >
+                            <a href={url} css={{ position: "relative", display: "block" }}>
```

Comment:
This should probably be a `<Link>` from `react-router`. An `a` reloads the entire page. I only think of this now, unfortunately; we should probably check existing code for this as well. :sweat_smile:

Reply:
Yeah good catch. I checked the rest of the code base: it's fine everywhere.

Reply:
Ok phew great thx. :sweat_smile: :relieved: